scripts/install_files.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$NO_COLOR = "\033[0m";
+$GREEN = "\033[38;5;010m";
+$YELLOW = "\033[38;5;011m";
+$ORANGE = "\033[38;5;214m";
+
+echo "\n${YELLOW}creating file for CSS personalization$NO_COLOR\n";
+touch('public/dist/user.css');
+
+echo "\n${YELLOW}creating default SQLite database$NO_COLOR\n";
+touch('database/database.sqlite');
+
+echo "\n${YELLOW}setting up hooks for git pull and git commits$NO_COLOR\n";
+copy('scripts/pre-commit', '.git/hooks/pre-commit');
+copy('scripts/post-merge', '.git/hooks/post-merge');
+
+echo "\n${ORANGE}To disable the call of composer and migration on pull add$NO_COLOR\n";
+echo "${ORANGE}a file named '.NO_AUTO_COMPOSER_MIGRATE' in this directory.$NO_COLOR\n";
```

Comment:
This comment is for the whole script not just this line. I am not sure, if this script works nicely on Windows. Obviously, nobody has ever being bothered by this question, because the old file was a `.sh`-script which obviously has never been working with Windows.

But now, after the script has become runnable for Windows we should maybe also ensure that it runs nicely. Here are the two things, I am not sure about:

 1. We use `\n` as the new line character. PHP offers the constant `PHP_EOL` which is `\n\r` on Windows, because Windows needs an additional carriage return. However, I do not know if this is only required for text files our for output to the command line, too. I would suspect the letter, because the programmer cannot know if the STDOUT targets the screen or is redirected to a file. So probably, we should use `PHP_EOL`.
 2. I am not sure, if Windows can handle the ANSI escape sequence and color codes.

Reply:
Great! Then we can keep the color codes. I feared that Windows would get confused in such a way that it would totally break console output. Glad to hear that it is the other way around!

But I still believe we need `PHP_EOL` instead of `\n`.

Moreover, another thought crossed my mind. I somehow remember that you figured out that the ANSI color codes are either only supported by `echo` or by `printf`. I don't remember what script you repaired and what the correct solution was, but have you checked that?